ci-operator/config/openshift/cluster-api-provider-baremetal/openshift-cluster-api-provider-baremetal-master.yaml

```diff
@@ -57,6 +57,16 @@ tests:
   commands: make vet
   container:
     from: src
+- as: manifests
+  commands: |
+    curl -s https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh | sh
+    PATH=.:$PATH make manifests
```

Comment:
Does this makefile target test anything? It looks like it will just produce the manifests.

Reply:
Good catch. This was fromm an old commit, which was supposed to be used rather than the generate-check script. I think its redundant and will be removed. Thanks.